protocol/avdl/keybase1/teams.avdl

```diff
@@ -1233,6 +1233,19 @@ protocol teams {
   // Note: allowInaction only applies to invite removals, just because that was what was needed.
   void teamRemoveMember(int sessionID, TeamID teamID, string username, string email, TeamInviteID inviteID, boolean allowInaction);
 
+  record RemoveMemberArgs {
```

Comment:
@mlsteele don't know if there's a better name for this

Reply:
`TeamMemberToRemove`